LR_Tourist/BLL/Services/ServiceService.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using BLL.Interfaces;
+using BLL.Model;
+using DA.Services.Repository;
+
+namespace BLL.Services
+{
+    public class ServiceService : IService<Service>, ICRUDService<Service>
```

Comment:
Looks very good)

Lets it will be ServiceManagementService.

PS: Maybe all services should end with <entity>ManagementService and ICRUDService should have name IEntityManagementService.